src/core/promgr/promgr.go

```diff
@@ -37,6 +37,8 @@ type ProjectManager interface {
 	Exists(projectIDOrName interface{}) (bool, error)
 	// get all public project
 	GetPublic() ([]*models.Project, error)
+	// get all projects that the user is authorized
+	GetAuthorized(user *models.User) ([]*models.Project, error)
```

Comment:
Is this a temp solution?

Long term I think it would be better to access a security Context, b/c in theory a security context that is not local may also has permission to access projects.